src/actions/get-bounty-canceled-event.ts

```diff
@@ -65,22 +62,11 @@ export async function action(block: DecodedLog, query?: EventsQuery): Promise<Ev
   const isHardCancel = ['open', 'ready'].includes(dbBounty?.state || '') && (dbBounty.fundingAmount === ('0' || undefined) || isFunded)
 
   if(isHardCancel) {
-    const [owner, repo] = slashSplit(dbBounty?.repository?.githubPath);
-
     if(dbBounty?.pull_requests.length > 0) 
-      closeAndRemovePullRequests(dbBounty?.pull_requests, owner, repo)
+      closeAndRemovePullRequests(dbBounty?.pull_requests)
```

Comment:
we can simply call pull_Request.destroy() here, we don't need to make a closure for this